docker/README.md

```diff
@@ -31,7 +31,8 @@ Ensure the the Styx configuration reads the origins from `/styx/config/` directo
 
 Finally, start the Styx Docker image using `/styx/config/styxconf.yml` as a configuration file:        
 
-    docker container run -d --name myStyx \
-        -p 8080:8080 -p 9000:9000 -p 8443:8443
+    docker container run -d  \
```

Comment:
maybe keep the name as in the previous examples and suggest --rm ?

Reply:
Changed the code as you suggested. Please have a look again.